Tests/Library/LibraryTests.swift

```diff
@@ -22,3 +22,18 @@ struct LibraryTest {
         await #expect(color.red == 1.0)
     }
 }
+
+extension LibraryTest {
+    @Test func testCallbackOperation() async {
```

Comment:
these "sometimes inline" really frustrate me personally tbh, there's no formatter out there which supports this rule... Can we revisit soon please?

Reply:
Ah I think I understand:

```swift
@Test func testCallbackOperation() async {}
```

vs

```swift
@Test
func testCallbackOperation() async {}
```

I have also been confused about which style to use when. I was trying to follow the conventions I find. swift-testing uses both styles in their readme., but always uses inline with plain `@Test`...